x/delayedack/keeper/fraud.go

```diff
@@ -18,10 +18,14 @@ func (k Keeper) HandleFraud(ctx sdk.Context, rollappID string) error {
 	logger.Debug("Reverting IBC rollapp packets", "rollappID", rollappID)
 	for _, rollappPacket := range rollappPendingPackets {
 		errString := "fraudulent packet"
-		packetId := channeltypes.NewPacketID(rollappPacket.Packet.GetDestPort(), rollappPacket.Packet.GetDestChannel(), rollappPacket.Packet.GetSequence())
-		logger.Debug("Reverting IBC rollapp packet", "rollappID", rollappID, "packetId", packetId, "type", rollappPacket.Type)
+		logger.Debug("Reverting IBC rollapp packet", "rollappID", rollappID,
+			"type", rollappPacket.Type,
+			"source channel", rollappPacket.Packet.SourceChannel,
+			"sequence", rollappPacket.Packet.Sequence)
 
 		if rollappPacket.Type == commontypes.RollappPacket_ON_RECV {
+			// PacketID is only guaranteed to be unique in case the packet is of type OnRecvPacket
+			packetId := channeltypes.NewPacketID(rollappPacket.Packet.GetDestPort(), rollappPacket.Packet.GetDestChannel(), rollappPacket.Packet.GetSequence())
```

Comment:
Not sure, but maybe you wanted to use `NewPacketUID` here?

Reply:
yea. it's just for logging purposes and currently @mtsitrin is changing this part anyway so to prevent him from conflicts will keep it like that.